### ISSUE TITLE
Issue with k8s.io/docs/concepts/workloads/controllers/deployment/

### ISSUE THREAD

**Issue by tigran-a** [kubernetes/website#9505]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
In section "Rolling Back a Deployment"  of https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

It seems that nginx-deployment-3066724191 should have 0 READY, as its pods are in a failed state
(and nginx-deployment-1564180365 should have 2 READY)

```
$ kubectl get rs
NAME                          DESIRED   CURRENT   READY   AGE
nginx-deployment-1564180365   2         2         0       25s
nginx-deployment-2035384211   0         0         0       36s
nginx-deployment-3066724191   2         2         2       6s
```
Looking at the Pods created, you will see that the 2 Pods created by new ReplicaSet are stuck in an image pull loop.
```
$ kubectl get pods
NAME                                READY     STATUS             RESTARTS   AGE
nginx-deployment-1564180365-70iae   1/1       Running            0          25s
nginx-deployment-1564180365-jbqqo   1/1       Running            0          25s
nginx-deployment-3066724191-08mng   0/1       ImagePullBackOff   0          6s
nginx-deployment-3066724191-eocby   0/1       ImagePullBackOff   0          6s
```

**Proposed Solution:**
(described in the previous section)
```
$ kubectl get rs
NAME                          DESIRED   CURRENT   READY   AGE
nginx-deployment-1564180365   2         2         2       25s
nginx-deployment-2035384211   0         0         0       36s
nginx-deployment-3066724191   2         2         0       6s
```

**Page to Update:**
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

**Comment by neolit123** [comment]
@kubernetes/sig-apps-bugs 
@kubernetes/sig-cli-bugs 

/kind bug

**Comment by csskuber** [comment]
related PR #9584

**Comment by mattfarina** [comment]
This appears to be closed because of the merger of #9584. If there is still an issue we can re-open this.

/close
